tests/specs/receive/receive.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { test } from '../../fixtures/fixtures';
+
+test.describe('Receive Dialog', () => {
```

Comment:
Very basic test to check the correct tokens are listed. To be extended with more tests to assert clicking QR code renders the correct UI